clusterloader2/testing/density/config.yaml

```diff
@@ -0,0 +1,128 @@
+automanagedNamespaces: 3
+steps:
+- measurements:
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: TimerCreate
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: SchedulerLatency
+    Method: SchedulerLatency
+    Params:
+      action: reset
+      provider: gce
+      masterName: e2e-ksiedlecki2-master
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+- phases:
+  - namespaceRange:
+      min: 0
```

Comment:
You can change in separate PR. But I think it's better user experience if the indexing for namespace range is 1-based rather than 0-based.

Reply:
done.